snuba/web/rpc/v1/resolvers/R_eap_spans/common/sampling_in_storage_util.py

```diff
@@ -157,9 +162,16 @@ def run_query_to_correct_tier(
             _get_time_budget() / 1000,
         )
         query_settings.push_clickhouse_setting("timeout_overflow_mode", "break")
-        target_tier = _get_target_tier(timer)
+        target_tier, estimated_target_tier_query_duration = _get_target_tier(
+            timer, metrics_backend, referrer
+        )
 
-        if target_tier == Tier.TIER_512:
+        if target_tier == _get_most_downsampled_tier():
+            metrics_backend.timing(
+                "sampling_in_storage_difference_between_estimated_and_true_query_duration",
+                0,
```

Comment:
0 ?

Reply:
I would say in this case don't record it at all, it'll just mess up the reporting. in this case the estimate was not used